distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```
@@ -274,15 +316,41 @@ type Connection
          representing the query to execute.
     execute_update : Text | SQL_Statement -> Integer
     execute_update self query =
-        statement_setter = self.dialect.get_statement_setter
-        self.jdbc_connection.with_prepared_statement query statement_setter stmt->
-            Panic.catch UnsupportedOperationException stmt.executeLargeUpdate _->
-                stmt.executeUpdate
+        if Execution_Context.Output.is_enabled.not then Error.throw (Forbidden_Operation.Error "Executing update queries is forbidden as the Output context is disabled.") else
```

Lets fold this into the `if_enabled` method. 

We just need to provide the message and pass in.

---

But `if_enabled` panics (and IMO rightfully so), but here we want to return a dataflow error.

---

we can put it easily in as `~on_disabled=(Panic.throw (Forbidden_Operation.Error self.name))` argument.

---

Ok, but at this point I don't see how is that simpler than this `if`.